addons/im_livechat/static/src/models/public_livechat_global.js

```diff
@@ -83,6 +83,15 @@ registerModel({
             }
             return clear();
         },
+        _computeWelcomeMessages() {
+            const welcomeMessages = this.messages.filter((message) => {
+                return message.id && typeof message.id === 'string' && message.id.startsWith('_welcome_');
+            });
+            if (welcomeMessages.length === 0) {
+                return clear();
+            }
```

Comment:
this is not useful, many relations accept replace empty array

Reply:
change done